shopfloor/actions/search.py

```diff
@@ -28,11 +30,20 @@ def package_from_scan(self, barcode):
             return model.browse()
         return model.search([("name", "=", barcode)], limit=1)
 
-    def picking_from_scan(self, barcode):
+    def picking_from_scan(self, barcode, accept_source_document=False):
```

Comment:
We have the same kind of parameter named `use_packaging` when scanning a product, so I would name it the same, and based on the technical field name, not its label:
```suggestion
    def picking_from_scan(self, barcode, use_origin=False):
```